docs/deployment/README.md

```diff
@@ -52,13 +52,13 @@ For other cases,
 are available.
 
 ```shell
- $ tfenv install 0.12.12
-[INFO] Installing Terraform v0.12.12
+ $ tfenv install 0.14.4
```

Comment:
Worth noting: 14.5 is out.

Reply:
Resolving since we're not going to do the upgrade to 0.14